### PR TITLE
meatball menu!

### PR DESCRIPTION
**Pull request by sampoder**
Screenshot 2024-03-09 at 6 03 56 PM

### CODE REVIEW

File: app/views/hcb_codes/_meatballs.html.erb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      <%= hcb_code.pin.present? ? "Unpin from" : "Pin to" %> <%= hcb_code.event.name %>
+    <% end %>
+    <% if (hcb_code.stripe_card? || hcb_code.stripe_force_capture?) && hcb_code.stripe_card.present? && !hcb_code.personal_transaction.present? %>
+      <div class="menu__divider"></div>
+      <%= link_to invoice_as_personal_transaction_hcb_code_path(id: hcb_code.hashid), method: :post, class: "tooltipped tooltipped--s flex items-center", 'aria-label': "Invoice #{hcb_code.stripe_cardholder&.user&.name || "yourself"} for this transaction.", data: { confirm: "Are you sure you'd like to invoice #{hcb_code.stripe_cardholder&.user&.name || "yourself"} for this transaction? They'll be emailed an invoice." } do %>
+        <%= inline_icon "sad", size: 24 %>
+        Flag as personal transaction
+      <% end %>
+    <% end %>
+    <% if (hcb_code.stripe_card? || hcb_code.stripe_force_capture?) && hcb_code.stripe_card.present? %>
```
Comment (garyhtou) [inline app/views/hcb_codes/_meatballs.html.erb]: We should use [`can_dispute?`](https://github.com/hackclub/bank/blob/9e4a7bcb4466fceb9ba1a0555f04321af9285067/app/services/hcb_code_service/can_dispute.rb#L4) here.

Comment (garyhtou) [reply app/views/hcb_codes/_meatballs.html.erb]: Turns out there is a reason for making transactions such as donations disputable (see the code).

**Review by garyhtou — COMMENTED** [review]
Is there a way to get the top & right margins to be equal?
image

Besides that, I've just got some small feedback.

🪄 Merge wand is your!

**Comment by garyhtou** [comment]
🧆 quite tasty!

File: app/views/hcb_codes/_meatballs.html.erb (hunk 1 above)
Comment (sampoder) [reply app/views/hcb_codes/_meatballs.html.erb]: interesting...

**Comment by garyhtou** [comment]
image

**Pull request merged**